lldb/source/Target/Process.cpp

```diff
@@ -5930,6 +5930,11 @@ void Process::PrintWarningCantLoadSwiftModule(const Module &module,
 }
 
 void Process::PrintWarningToolchainMismatch(const SymbolContext &sc) {
+  if (GetTarget().GetProcessLaunchInfo().IsScriptedProcess())
+    // It's very likely that the debugger used to launch the ScriptedProcess
+    // will not match the compiler version used to build the target, so we
+    // should just silence the toolchain mismatch warning.
```

Comment:
This comment doesn't really explain why this necessary. I would explain that there's no need for the debugger and swift compiler to match, because symbolicating a crashlog doesn't require the serialized AST. I would also explain why it's safe to silence this warning: because it doesn't depend on us using that, but instead this function is called from a relatively generic place (`FrameSelectedCallback`).